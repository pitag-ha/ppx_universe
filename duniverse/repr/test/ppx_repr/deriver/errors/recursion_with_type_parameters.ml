type 'a t1 = 'a * t2 option

and t2 = int * unit t1 option [@@deriving repr]
