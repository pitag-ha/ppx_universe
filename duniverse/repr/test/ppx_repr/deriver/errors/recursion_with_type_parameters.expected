Fatal error: exception (Failure "Can't support mutually-recursive types with type parameters")
